test/it/fileutils/mv_gnu_test.sh

```
@@ -0,0 +1,29 @@
+    printf 'A\n' > ${TEST_DIR}/a.txt
+    printf 'B\n' > ${TEST_DIR}/b.txt
+}
+
+CleanUp() {
+    rm -rf ${MIMIXBOX_IT_ROOT}/mv_gnu
+}
+
+# mv --target-directory dst a b moves each source into dst (destination-first).
+TestMvTargetDirectory() {
+    export TEST_DIR=${MIMIXBOX_IT_ROOT}/mv_gnu
+    mv --target-directory ${TEST_DIR}/dst ${TEST_DIR}/a.txt ${TEST_DIR}/b.txt
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)